css-brochure-form/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Brochure Form</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <h2>LEARN MORE</h2>
+  <form>
+    <div>
+      <label for="email">COMPANY EMAIL</label>
+    </div>
+    <input type="text" id="email" placeholder="sally@company.co">
+    <div>
+      <label for="name">NAME</label>
+    </div>
+    <input type="text" id="name" placeholder="Sally Smith">
+    <div>
+      <label for="wherehire">WHERE ARE YOU HIRING?</label>
+    </div>
+    <input type="text" id="wherehire" placeholder="City, Country">
+    <div>
+      <label for="engineershiring">HOW MANY ENGINEERS ARE YOU HIRING IN THE NEXT 6 MONTHS?</label>
+    </div>
+    <select name="engineers" id="engineershiring">
+      <option value="1-2 Engineers">1-2 Engineers</option>
+      <option value="2-3 Engineers">2-3 Engineers</option>
+      <option value="3-4 Engineers">3-4 Engineers</option>
+      <option value="4-5 Engineers">4-5 Engineers</option>
+    </select>
+    <div>
+      <input type="submit" name="submitbutton" value="SUBMIT">
```

Comment:
Try css rule to update submit button text style instead of using `value="SUBMIT"`